src/Files.App/Strings/en-US/Resources.resw

```diff
@@ -4325,4 +4319,7 @@
   <data name="FailedToOpenLogFile" xml:space="preserve">
     <value>Unable to open the log file</value>
   </data>
+  <data name="Recent" xml:space="preserve">
```

Comment:
Renaming the string will require users to retranslate it. I would leave the name as is unless there is a reason to change it.